examples/config-dev.yaml

```diff
@@ -65,7 +66,10 @@ staticClients:
   - 'http://127.0.0.1:5555/callback'
   name: 'Example App'
   secret: ZXhhbXBsZS1hcHAtc2VjcmV0
-
+#  - id: example-device-client
+#    redirectURIs:
+#      - /dex/device/callback
+#    name: 'Static Client for Device Flow'
```

Comment:
For the device flow to work properly with redirect URIs, I set up my device clients to register its device callback as its valid redirect URI